spec/types/time_value_spec.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+RSpec.describe Bali::Types::TimeValue do
+  describe 'integration with a model' do
+    it 'casts a time value from integer to a date string' do
+      workout = Workout.new(workout_start_at: 3600)
+      expect(workout.workout_start_at).to eql("#{Date.current} 01:00:00")
+    end
+
+    it 'serializes a date string to a number in seconds' do
+      workout = Workout.create(workout_start_at: "#{Date.current} 01:00:00")
+      stored_value = workout.read_attribute_before_type_cast(:workout_start_at)
+      expect(stored_value).to eql(3600)
+    end
+  end
+
+  subject { Bali::Types::TimeValue.new }
+
+  describe '#cast' do
+    it 'return a default time if value is blanck' do
+      expect(subject.cast('')).to eql("#{Date.current} 00:00:00")
+    end
+
+    it 'returns value if is a date string' do
+      date_string = "#{Date.current} 00:00:00"
```

Comment:
```suggestion
      date_string = "#{Date.current} 00:00:00"
```
This is a DateTime string, not date.